helm/prometheus-agent/charts/prometheus-agent/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    {{ include "labels.common" . | nindent 4 }}
+  name: {{ include "name" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/instance: {{ include "name" . }}
+      app.kubernetes.io/name: prometheus
+  egress:
+  - {}
+  ingress:
```

Comment:
Do we need to allow ingress traffic ?

Reply:
I'm not sure but let's try with a broad one and restrict later